client/Lykke.Service.ChainalysisProxy.Client/Lykke.Service.ChainalysisProxy.Client.xml

```diff
@@ -4,5 +4,4473 @@
         <name>Lykke.Service.ChainalysisProxy.Client</name>
     </assembly>
     <members>
+        <member name="T:Lykke.Service.ChainalysisProxy.Client.AutorestClientMapper">
```

Comment:
Delete and ignore the file in Git. It is autogenerated.

Reply:
Done